w2/server/http/product_handler.go

```diff
@@ -41,8 +43,8 @@ func (s *HttpServer) handleProductEdit(w http.ResponseWriter, r *http.Request) {
 	}
 
 	payload.ID = chi.URLParam(r, "productId")
-	_, err := s.productService.UpdateProduct(payload)
-	if err != nil {
+	_, err := s.productService.UpdateProduct(payload, userID)
+	if err != (commons.CustomError{}) {
```

Comment:
- CustomError impelement HTTPStatusCode interface and handleError has handled for that error, so we don't need check the error anymore.